README.md

```diff
@@ -31,56 +31,62 @@
 
 ## Overview
 
-[Stockfish][website-link] is a **free and strong UCI chess engine** derived from
-Glaurung 2.1 that analyzes chess positions and computes the optimal moves.
+[Stockfish][website-link] is a **free UCI chess engine**, derived from
+the Glaurung 2.1 chess engine, to analyze chess positions and also to
+compute optimal chess moves with it.
 
-Stockfish **does not include a graphical user interface** (GUI) that is required
-to display a chessboard and to make it easy to input moves. These GUIs are
-developed independently from Stockfish and are available online. **Read the
-documentation for your GUI** of choice for information about how to use
-Stockfish with it.
+Stockfish **does not include** a graphical user interface (GUI), which
+are required to display a chessboard and to input chess moves. They are
```

Comment:
`Stockfish **does not include** a graphical user interface (GUI), which
are required to display a chessboard`
why refer to 'a graphical user interface' as plural `which are` instead of 'that is' is it an English Grammar thing?

Reply:
Yeah, this was a total oversight. Thank you for the heads up. It is fixed now.